.changelog/unreleased/bug-fixes/2311-register-wasm-v1beta1-types.md

```diff
@@ -0,0 +1 @@
+* register legacy CosmWasm v1beta1 MsgExecuteContract to support older tx decoding [#2311](https://github.com/provenance-io/provenance/issues/2311).
```

Comment:
Please capitalize the first character since changelog entries are supposed to be sentences.